api/api.go

```diff
@@ -399,7 +400,8 @@ func CreateOrb(ctx context.Context, logger *logger.Logger, name string, namespac
 }
 
 // OrbSource gets the source or an orb
-func OrbSource(ctx context.Context, logger *logger.Logger, name string) (string, error) {
+func OrbSource(ctx context.Context, logger *logger.Logger, namespace string, orb string) (string, error) {
```

Comment:
Code consistency:

Is an `orb` `namespace/name` or is a `name` `namespace/orb`? (CreateOrb uses `namespace/name`, but this uses `namespace/orb`, which will be potentially confusing to future contributors)

Reply:
That's a good point, and one inconsistency we should fix before shipping.

I will make a follow up PR to fix this though.

FWIW, they should all be `orb [command] NAMESPACE ORB` as we don't want to start inventing a uri-like syntax.